wikimedia/wikipedia-ios
# Diffs - Add share functionality back into diff view

**Phabricator:** https://phabricator.wikimedia.org/T335579 (partially)

### Notes
- Adds share functionality back into the diff view, via the more > share button
- Also adds edit history icon and delegate method (still need to push actual `PageHistoryViewController` when tapped)

### Test Steps
1. View edit history for an article, and tap a diff
2. Tap the more > share button in the toolbar
3. Confirm shared link opens expected revision(s) when viewed on web
